test/jdk/tools/pack200/PackTestZip64.java

```diff
@@ -85,43 +91,62 @@ static void testPacking() throws IOException {
         cmdsList.add(unpackedFile.getName());
         Utils.runExec(cmdsList);
 
-        // Compare files binary
-        compareTwoFiles(testFile, unpackedFile);
+        if (!testFile.exists()) {
+            throw new IOException("File " + testFile.getName() + " does not exist!");
+        }
+
+        if (!unpackedFile.exists()) {
+            throw new IOException("File " + unpackedFile.getName() + " does not exist!");
+        }
+        // Compare two Jar files
+        compareTwoJarFiles(new JarFile(testFile), new JarFile(unpackedFile));
 
         // Cleaning up generated files
         testFile.delete();
         packedFile.delete();
         unpackedFile.delete();
     }
 
-    static void compareTwoFiles(File src, File dst) throws IOException {
-        if (!src.exists()) {
-            throw new IOException("File " + src.getName() + " does not exist!");
-        }
-
-        if(!dst.exists()) {
-            throw new IOException("File " + dst.getName() + " does not exist!");
-        }
+    static void compareTwoJarFiles(JarFile srcJar, JarFile dstJar) throws IOException {
+        for (Enumeration srcEntries = srcJar.entries(); srcEntries.hasMoreElements(); ) {
+            JarEntry srcEntry = (JarEntry) srcEntries.nextElement();
+            JarEntry dstEntry = dstJar.getJarEntry(srcEntry.getName());
+            if (dstEntry == null) {
+                throw new IOException("Jar Entry "+srcEntry.getName()+" does not exist in "+dstJar.getName());
```

Comment:
Please insert a space on each side of all binary operators:
```
                throw new IOException("Jar Entry " + srcName + " does not exist in " + dstJar.getName());
```

Reply:
Fixed in https://github.com/ibmruntimes/openj9-openjdk-jdk11/compare/a16261f995e7aaf04f90db78cbaf9188c987c6a9..c90e742fe22a0d8936317c6974ab336ec14f361f